src/sentry/api/endpoints/project_stacktrace_link.py

```diff
@@ -258,6 +260,35 @@ def get(self, request: Request, project: Project) -> Response:
                     # When no code mapping have been matched we have not attempted a URL
                     if current_config["outcome"].get("attemptedUrl"):
                         result["attemptedUrl"] = current_config["outcome"]["attemptedUrl"]
+
+                should_get_codecov_line_coverage = (
+                    features.has(
+                        "organizations:codecov-stacktrace-integration",
+                        project.organization,
+                        actor=request.user,
+                    )
+                    and project.organization.flags.codecov_access
+                )
+                if should_get_codecov_line_coverage:
+                    try:
+                        (
+                            result["lineCoverage"],
+                            result["codecovStatusCode"],
+                        ) = Codecov.get_codecov_line_coverage(
+                            repo=current_config["config"]["repoName"],
+                            service=current_config["config"]["provider"]["key"],
+                            branch=current_config["config"]["defaultBranch"],
+                            path=current_config["outcome"]["sourcePath"],
+                        )
+                    except Exception:
```

Comment:
If we use raise_status(), this code would look more like this:
```
                    except requests.exceptions.HTTPError as error:
                        if error.response.status_code == 404:
                            logger.exception("Failed to get expected coverage data from Codecov, pending investigation. Continuing execution.")
                    except Exception:
                        logger.exception("Something unexpected happen. Continuing execution")
```

For now, let's error on everything and later on determine if we need to silence certain cases:
```
                    except Exception:
                        logger.exception("Failed to get expected coverage data from Codecov, pending investigation. Continuing execution.")
```

You can remove the next few lines.